components/page.js

```diff
@@ -12,9 +18,57 @@ const Content = styled.div`
   padding: 60px;
 `;
 
-export default ({ hackathons, currentPath, children }) => (
-  <Container>
-    <Sidebar currentPath={currentPath} hackathons={hackathons} />
-    <Content>{children}</Content>
-  </Container>
-);
+const HackathonNavBar = styled.div`
+  display: flex;
+  padding-left: 60px;
+`;
+
+const NavItem = styled.a`
+  display: block;
+  cursor: pointer;
+  font-size: 24px;
+  margin-right: 40px;
+  text-decoration: none;
+  color: ${p => (p.selected ? COLOR.BLACK : COLOR.DARK_COPY)};
+  border-bottom: ${p => (p.selected ? '3px solid black' : 'none')};
+`;
+
+export default ({ hackathons, currentPath, children }) => {
+  const [currPath, setCurrPath] = useState('intro');
+  useEffect(() => {
+    setCurrPath(window.location.pathname.split('/')[2]);
+  }, []);
+
+  return (
+    <Container>
+      <Sidebar currentPath={currentPath} hackathons={hackathons} />
+      <div>
+        <HomeHeader>{currentPath} Home</HomeHeader>
+        <HackathonNavBar>
+          <NavItem
+            key={`${currentPath}intro`}
+            href={`/${currentPath}/intro`}
+            selected={currPath === 'intro'}
+          >
+            Intro
+          </NavItem>
+          <NavItem 
+            key={`${currentPath}faq`} 
+            href={`/${currentPath}/faq`}
+            selected={currPath === 'faq'}
+          >
+            FAQ
+          </NavItem>
+          <NavItem
+            key={`${currentPath}sponsors`}
+            href={`/${currentPath}/sponsors`}
+            selected={currPath === 'sponsors'}
+          >
+            Sponsors
+          </NavItem>
```

Comment:
Could you use a [Next/Link](https://nextjs.org/docs/api-reference/next/link) instead of a <a> tag? It will make the transition between pages much smoother since it won't reload the whole site.

Reply:
![image](https://user-images.githubusercontent.com/45786074/91902649-e87bee00-ec56-11ea-8417-ba90c2340c2c.png)

this error is in the console briefly before the page reloads entirely even while using next/link hmm

Reply:
hmm maybe instead of using a next/link we can just use the router. Can you try using [router.push ](https://nextjs.org/docs/api-reference/next/router) instead?